www/experiments/lcp.inc

```diff
@@ -74,8 +75,9 @@ if (isset($lcp)) {
                 "expval" => array($lcpSource . "|as_image"),
                 "explabel" => array($lcpSource)
             ];
+            
             // priority Hints only help for foreground images
-            if (!$isBG && !$isVideo && !(isset($lcp['element']['fetchPriority']) && $lcp['element']['fetchPriority'] == 'high')) {
+            if (!$isBG && !$isVideo && !$hasHighFetchPriority) {
```

Comment:
This is getting close to breaking out into its own variable or changing into something like 

```php
$needs_priority_hints = !($isBG || $isVideo || $hasHighFetchPriority)
```

Reply:
hmm yeah nice idea. maybe `$suggest_priority_hints = ` ?